unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationHUD/NotificationHUDView.cs

```diff
@@ -48,4 +36,8 @@ public Notification ShowNotification(Model notificationModel)
     private void OnNotificationDismissed(INotification notification) { OnNotificationDismissedEvent?.Invoke(notification); }
 
     public void SetActive(bool active) { gameObject.SetActive(active); }
-}
\ No newline at end of file
+
+    public void Dispose()
+    {
+    }
```

Comment:
Why do we need to add an empty Dispose?

Reply:
I think, because of Factory where it used. It requires this interface